packages/@react-aria/visually-hidden/docs/VisuallyHidden.mdx

```diff
@@ -59,3 +59,17 @@ let {visuallyHiddenProps} = useVisuallyHidden();
 
 <div {...visuallyHiddenProps}>I am hidden</div>
 ```
+
+## Gotchas
+
+VisuallyHidden is positioned absolutely, so it needs a positioned ancestor, otherwise it will be positioned relative to the nearest positioned ancestor which may be the body, causing undesired scroll bars to appear.
```

Comment:
```suggestion
VisuallyHidden is positioned absolutely, so it needs a positioned ancestor. Otherwise, it will be positioned relative to the nearest positioned ancestor, which may be the body, causing undesired scroll bars to appear.
```

just some grammar stuff